spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+RSpec.feature 'Post index page', type: :feature do
+  before do
+    @user = User.find_by(name: 'Tom')
+  end
+
+  scenario 'I can see how many comments a post has.' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@user.posts.first.comments_counter)
+  end
+
+  scenario 'I can see how many likes a post has.' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@user.posts.first.likes_counter)
+  end
+
+  scenario 'I can see a section for pagination if there are more posts than fit on the view.' do
+    visit user_posts_path(@user)
+    expect(page).to have_css('.pagination')
+  end
+
+  scenario 'When I click on a post, I am redirected to that post\'s show page.' do
+    visit user_posts_path(@user)
+    first('.post-container').click
+    expect(page).to have_current_path(user_post_path(@user, @user.posts.first))
+  end
+end
```

Comment:
- [ **OPTIONAL** ] As the previous reviewer didn't mention the bellow tests asserts, kindly ensure that you also add **Test(s) that asserts you can see some of the post's body.** for the project requirements:

- [ **OPTIONAL** ] Also for best practices, I think it would be better if you put all tests regarding `post_index_spec` in one file instead of two different files. Right now, they are in 2 file: `spec/features/post_index_spec.rb` and `spec/features/post_index.users_spec.rb`

Reply:
@tresorsawasawa 
I'm sorry, I couldn't put all the tests and I deleted some redundant tests because Rubocop showed me an error telling me that there were too many lines. What can I do?